### ISSUE TITLE
_disableAccessibilityState

### ISSUE THREAD

**Issue by danielghost** [adaptlearning/adapt-contrib-core#186]
It looks like this attribute is missing from the core schemas except for _component.model.schema_.

**adapt-contrib-boxMenu:**
Should `_disableAccessibleState` be changed to `_disableAccessibilityState` for consistency with other models. Also missing from the schema.

**Comment by oliverfoster** [comment]
removed in https://github.com/adaptlearning/adapt-contrib-core/pull/247
